Systems/engine.xml

```diff
@@ -584,5 +584,37 @@
             <property>/engines/active-engine/coughing</property>
         </output>
     </logic>
+    
+    <!-- ============================================================== -->
+    <!-- Carburetor Icing drops RPM                                     -->
+    <!-- ============================================================== -->
+
+    <filter>
+        <name>Engine 160 HP Carb Icing Factor</name>
+        <type>gain</type>
+        <input>
+            <condition>
```

Comment:
Here you have a conditional `<input`> (because of the `<condition>` in it), but there's no default `<input>` (one without a `<condition>`). What should happen if the condition is false? Just leave the output value at its current value, or set it to 0? I think you want the latter, and in that case you should add

```xml
<input>
    <value>0.0</value>
</input>
```

after the first `<input>` block.